src/main/java/com/exasol/projectkeeper/files/ProjectFilesFixer.java

```diff
@@ -65,7 +65,7 @@ private void writeFile(final File projectFile, final InputStream template) {
                 projectFile.mkdirs();
                 Files.copy(template, projectFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
             } catch (final IOException exception) {
-                this.log.error("Failed to create or replace " + projectFile.getAbsolutePath());
+                this.log.error("E-PK-16 Failed to create or replace " + projectFile.getAbsolutePath());
```

Comment:
I think it's better to add `:` after the code, so the messages look the same in all our projects.

Reply:
:+1: